contrib/for-tests/netexec/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM busybox
+MAINTAINER Abhishek Shah "abshah@google.com"
+
+ADD netexec netexec
```

Comment:
nit: I like to add the .go file in these cases as an extra way of publishing what is happening.  Up to you.